internal/reconcilerevent/cluster.go

```diff
@@ -48,6 +48,18 @@ func (s *PeerRemovedEvent) Record(recorder record.EventRecorder) {
 		fmt.Sprintf("Removed EtcdPeer with name %q", s.PeerName))
 }
 
+type PeerDecommissionedEvent struct {
+	Object   runtime.Object
+	PeerName string
+}
+
+func (s *PeerDecommissionedEvent) Record(recorder record.EventRecorder) {
+	recorder.Event(s.Object,
+		"Normal",
+		"PeerDecommissioned",
+		fmt.Sprintf("Decommissioned EtcdPeer with name %q", s.PeerName))
```

Comment:
From a user perspective, I'm uncertain whether the term 'decommissioned' is well defined enough for them to always know what it means. Perhaps rephrasing this to be "PeerDataRemoved", and then explicitly stating "The persistent volume claim for peer %q has been deleted" would be clearer?